apps/src/util/i18nStringTracker.js

```diff
@@ -0,0 +1,40 @@
+import experiments from '@cdo/apps/util/experiments';
+
+// A map of "string source" -> "list of strings used"
+// For example: window.stringUsageSources['maze'] = ['level_1.instruction.header']
+// The data recorded in this will periodically be uploaded.
+if (!window.stringUsageSources) {
+  // Initialize if it doesn't already exist.
+  window.stringUsageSources = {};
+}
+
+export default function localeWithI18nStringTracker(locale, source) {
+  if (!experiments.isEnabled(experiments.I18N_TRACKING)) {
```

Comment:
I'm not against it but why do we need an experiment here? We're not uploading anything yet, plus this flag needs a deploy to change.

Reply:
This is for protection in case this wrapper does some how screw things up when deployed to the wild. I did try testing on all the level types, but I'm trying to be extra careful since this is just a wide impacting change. I'd rather do another deploy to turn on the feature rather than an emergency deploy to turn it off.